server/filters.go

```diff
@@ -107,11 +107,12 @@ func pxcFilter(versions map[string]*pbVersion.Version, apply string, current str
 		}
 
 		for _, s := range sorted {
-			if s.Equal(c) || s.LessThan(c) {
+			if s.Equal(c) || (s.Major() < c.Major() && s.Minor() < c.Minor() && s.Patch() < c.Patch()) {
```

Comment:
maybe will it be better?
```suggestion
			if s.Major() = c.Major() && s.Minor() = c.Minor() && s.Patch() = c.Patch() {
```